packages/core/src/metrics/envelope.ts

```diff
@@ -30,7 +30,7 @@ export function createMetricEnvelope(
   return createEnvelope<StatsdEnvelope>(headers, [item]);
 }
 
-function createMetricEnvelopeItem(metricBucketItems: Array<MetricBucketItem>): StatsdItem {
+function createMetricEnvelopeItem(metricBucketItems: MetricBucketItem[]): StatsdItem {
```

Comment:
so nit, I always prefer `Array<T>` because of https://tkdodo.eu/blog/array-types-in-type-script, but this is fine as well.

Maybe we should enforce one vs. other with lint rule.